### ISSUE TITLE
Pretrained models for MS-SSIM

### ISSUE THREAD

**Issue by Yuval-H** [LiuLei95/PyTorch-Learned-Image-Compression-with-GMM-and-Attention#5]
Thank you for the great code!

Would you happen to have any pretrained models weights trained with MS-SSIM loss?

**Comment by LiuLei95** [comment]
We don't have any pretrained models trained with MS-SSIM loss.
